test/api/test_tools.py

```diff
@@ -440,7 +440,6 @@ def test_run_by_versions(self):
             self.assertEqual(output1_content.strip(), "Version " + version)
 
     @skip_without_tool("multiple_versions")
-    @uses_test_history(require_new=False)
     def test_show_with_wrong_tool_version_in_tool_id(self, history_id):
```

Comment:
Well now that history_id will not have a value in this method definition.

Reply:
yeah, that's silly ...